source/connection.c

```diff
@@ -26,6 +28,17 @@
 #    pragma warning(disable : 4204) /* non-constant aggregate initializer */
 #endif
 
+static struct aws_http_connection_system_vtable g_default_connection_system_vtable = {
+    .new_socket_channel = aws_client_bootstrap_new_socket_channel,
+    .new_tls_socket_channel = aws_client_bootstrap_new_tls_socket_channel};
+
+static const struct aws_http_connection_system_vtable *g_aws_http_connection_system_vtable_ptr =
+    &g_default_connection_system_vtable;
```

Comment:
style nitpick: I like g_ too, but s_ is what we've been doing

also, just s_system_vtable would be much shorter, but that's just, like, my opinion man